plugins/modules/rds_cluster.py

```diff
@@ -1245,12 +1317,17 @@ def main():
             msg="skip_final_snapshot is False but all of the following are missing: final_snapshot_identifier"
         )
 
-    parameters = arg_spec_to_rds_params(dict((k, module.params[k]) for k in module.params if k in parameter_options))
     changed = False
+
+    parameters = arg_spec_to_rds_params(dict((k, module.params[k]) for k in module.params if k in parameter_options))
     method_name, method_options_name = get_rds_method_attribute_name(cluster)
 
     if method_name:
         if method_name == "delete_db_cluster":
+            if module.params.get("remove_from_global_db"):
+                if cluster["Engine"] in ["aurora", "aurora-mysql", "aurora-postgresql"]:
```

Comment:
This will thrown an error, when we try to delete a cluster that is not present.

Reply:
from my testing, doesn't seem to throw error.
but added `if cluster` for extra check